typescript/src/lib/arbitrum/l2-bitcoin-redeemer.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  EthersContractConfig,
+  EthersContractDeployment,
+  EthersContractHandle,
+} from "../ethereum/adapter"
+import { ArbitrumL2BitcoinRedeemer as L2BitcoinRedeemerTypechain } from "../../../typechain/ArbitrumL2BitcoinRedeemer"
+import { ChainIdentifier, Chains, L2BitcoinRedeemer } from "../contracts"
+import { EthereumAddress } from "../ethereum"
+import { Hex } from "../utils"
+import { BigNumber, Contract } from "ethers"
+
+import ArbitrumSepoliaL2BitcoinRedeemerDeployment from "./artifacts/arbitrumSepolia/ArbitrumL2BitcoinRedeemer.json"
+import ArbitrumSepoliaWormholeCoreDeployment from "./artifacts/arbitrumSepolia/WormholeCore.json"
+// TODO: Uncomment when Arbitrum L2BitcoinRedeemer is deployed
+// import ArbitrumWormholeCoreDeployment from "./artifacts/arbitrum/WormholeCore.json"
+
+/**
+ * Implementation of the Arbitrum L2BitcoinRedeemer handle.
+ * @see {L2BitcoinRedeemer} for reference.
+ */
+export class ArbitrumL2BitcoinRedeemer
+  extends EthersContractHandle<L2BitcoinRedeemerTypechain>
+  implements L2BitcoinRedeemer
+{
+  private readonly wormholeCore: Contract
+  private readonly recipientChain: number
+
+  constructor(config: EthersContractConfig, chainId: Chains.Arbitrum) {
+    let deployment: EthersContractDeployment
+    let wormholeCoreDeployment: EthersContractDeployment
+    let recipientChain: number
+
+    switch (chainId) {
+      case Chains.Arbitrum.ArbitrumSepolia:
+        deployment = ArbitrumSepoliaL2BitcoinRedeemerDeployment
+        wormholeCoreDeployment = ArbitrumSepoliaWormholeCoreDeployment
+        recipientChain = 10002 // Ethereum Sepolia
```

Comment:
Nitpick:
```suggestion
        recipientChain = ChainIds.ETHEREUM_SEPOLIA
```

Reply:
Since `recipientChain` here expects the Wormhole specific Chain Id (Not the same value as Ethereum Chain Id), I created a specific constant holding these values in: [refactor: add wormhole chain ids constant](https://github.com/threshold-network/tbtc-v2/pull/861/commits/06df45a311ebf242a24cae346d1365c77db79bcc)

For more info, please refer to:  https://wormhole.com/docs/products/reference/chain-ids/